localization/languages/cs-CZ.json

```diff
@@ -0,0 +1,152 @@
+        "suggestedSite": "Doporučené stránky", //this is used to label suggested websites from the DuckDuckGo API,
+        "resultsFromDDG": "Výsledky z DuckDuckGo", //this is used as a label to indicate which results come from DuckDuckGo's API
+        "taskN": "Dimenze %n", //this is used as a way to identify which tab a task is in "task 1", "task 2", ...
+        "hostsFileEntry": "Lokální soubor", //this is a label used to indicate that a result is coming from the user's host file        
+        /* custom !bangs
+        these are some of the items that show up when you press ! in the searchbar.
+        Each one of these strings describes what the command will do when you run it. */
+        "viewSettings": "Zobrazit nastavení",
+        "takeScreenshot": "Pořídit snímek obrazovky",
+        "clearHistory": "Vymazat celou historii",
+        "switchToTask": "Skočit do dimenze",
+        "createTask": "Vytvořit dimenzi",
+        "moveToTask": "Přesunout tuto kartu do dimenze",
+        "searchBookmarks": "Vyhledávat záložky",
+        /* navbar */
+        "enterReaderView": "Otevřít režim čtení",
+        "exitReaderView": "Zavřít režim čtení",
+        "emptyReadingListTitle": "Váš seznam čtení je prázdný.",
+        "emptyReadingListSubtitle": "Stránky, otevřené v režimu čtení, zde budou uloženy pro offline zobrazení po dobu 30 dnů.",
+        "newTabLabel": "Nová karta", //this is a noun, used for tabs that don't have a page loaded in them yet
+        "connectionNotSecure": "Vaše připojení k této stránce není zabezpečené.",
+        "searchbarPlaceholder": "Vyhledávejte nebo zadejte adresu",
+        "privateTab": "Soukromá karta",
+        "newTabAction": "Nová karta", //this is a verb, used to label a button that adds a tab to the tabstrip when clicked
+        /* task overlay */
+        "viewTasks": "Zobrazit dimenze",
+        "newTask": "Nová dimenze", //"new" is a verb - it is used for a button at the bottom of the task overlay
+        "defaultTaskName": "Dimenze %n", //this is the name used for newly-created tasks; %n is replaced with a number ("task 1", "task 2", etc)
+        /* find in page toolbar */
+        "searchInPage": "Vyhledávat na stránce", //this is used as the placeholder text for the textbox in the find in page toolbar
+        "findMatchesSingular": "%i z %t výsledku", //this and the next label are used to indicate which match is currently highlighted
+        "findMatchesPlural": "%i z %t výsledků",
+        /* Focus mode */
+        "isFocusMode": "Jste v režimu zaměření.",
+        "closeDialog": "OK", //used as a label for the button that closes the dialog
+        "focusModeExplanation1": "V režimu zaměření jsou všechny karty kromě aktuální skryté a nelze vytvářet nové karty.",
+        "focusModeExplanation2": "Můžete opustit režim zaměření odškrtnutím \"focus mode\" v nabídce zobrazení.",
```

Comment:
Should "focus mode" be translated here as well, since the actual menu item (```appMenuFocusMode```) is translated?

Reply:
Another good point.
This should be automatic replaced in program! ;)